modules/presence/subscriber.c

```diff
@@ -302,6 +341,8 @@ int subscriber_init(void)
 
 	info("Subscribing to %u contacts\n", list_count(&presencel));
 
+	contact_set_update_handler(contacts, contact_handler, (void*)0x1337);
+
```

Comment:
this function must also be called with a NULL handler from `subscriber_close`.

otherwise, after the module has been unloaded, the `contact.c` may still call
a function that does not exist (will lead to a crash).